code/game/objects/closets/walllocker.dm

```diff
@@ -54,6 +54,27 @@
 	pixel_x = WORLD_ICON_SIZE
 	dir = EAST
 
+/obj/structure/closet/walllocker/emerglocker/snow
+	spawnitems = list(/obj/item/weapon/storage/toolbox/emergency/snow)
```

Comment:
Why does this one have snow toolboxes and the regular closet have regular toolboxes?